ui/app/AppLayouts/Chat/popups/community/CommunityDetailPopup.qml

```diff
@@ -15,19 +15,19 @@ StatusModal {
     id: root
 
     property var store
-    // Not Refactored Yet
-    property QtObject community: null // root.store.chatsModelInst.communities.observedCommunity
+    property QtObject community: root.store.communitiesModuleInst.observedCommunity
     property string communityId: community.id
     property string name: community.name
     property string description: community.description
     property int access: community.access
-    property string source: community.thumbnailImage
-    property int nbMembers: community.nbMembers
+    property string source: community.image || ""
```

Comment:
@jrainville please remove this check, entire logic is set from the backend side, since `community.image` is already a string on the backend side it will contain image or be an empty string, by default it's already an empty string. So just this is enough:
`property string source: community.image`